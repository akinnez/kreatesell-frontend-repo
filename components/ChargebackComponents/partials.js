export const dayOptions = [
  { value: "Today", label: "Today" },
  { value: "Yesterday", label: "Yesterday", unit: 1 },
  { value: "Last 7 days", label: "Last 7 days", unit: 7 },
  { value: "Last 30 days", label: "Last 30 days", unit: 30 },
  { value: "This year", label: "This year", unit: 365 },
  //   { value: "All time", label: "All time" },
];

export const ticketsOptions = [
  { value: "All Tickets", label: "All Tickets" },
  { value: "Opened Tickets", label: "Open Tickets" },
  { value: "Closed Tickets", label: "Closed Tickets" },
];

export const currencyOptions = [
  { value: "Nigeria", label: "NGN" },
  { value: "USA", label: "USD" },
  { value: "Spain", label: "GBP" },
  { value: "Spain", label: "KES" },
  { value: "Spain", label: "ZAR" },
  { value: "Spain", label: "TZX" },
  { value: "Spain", label: "UGX" },
];
