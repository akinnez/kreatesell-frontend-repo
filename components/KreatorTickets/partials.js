export const dayOptions = [
  { value: "Today", label: "Today" },
  { value: "Yesterday", label: "Yesterday", unit: 1 },
  { value: "Last 7 days", label: "Last 7 days", unit: 7 },
  { value: "Last 30 days", label: "Last 30 days", unit: 30 },
  { value: "This year", label: "This year", unit: 365 },
  //   { value: "All time", label: "All time" },
];

export const ticketsOptions = [
  { value: "All Tickets", label: "All Tickets" },
  { value: "Opened Tickets", label: "Open Tickets" },
  { value: "Closed Tickets", label: "Closed Tickets" },
];

// export const affiliateOptions = [
//   { value: "/account/dashboard/affiliate", label: "Affiliate Dashboard" },
//   { value: "Quick Stats", label: "Quick Stats" },
//   { value: "Find Products", label: "Find Products" },
// ];
