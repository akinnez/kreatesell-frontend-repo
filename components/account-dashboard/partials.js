export const dayOptions = [
	{ value: "Custom", label: "Custom" },
	{ value: "Today", label: "Today" },
	{ value: "Yesterday", label: "Yesterday" },
	{ value: "Last 7 days", label: "Last 7 days" },
	{ value: "Last 30 days", label: "Last 30 days" },
	{ value: "This year", label: "This year" },
	{ value: "All time", label: "All time" },
];

export const currencyOptions = [
	{ value: "Nigeria", label: "NGN" },
	{ value: "USA", label: "USD" },
	{ value: "Spain", label: "GBP" },
	{ value: "Spain", label: "KES" },
	{ value: "Spain", label: "ZAR" },
	{ value: "Spain", label: "TZX" },
	{ value: "Spain", label: "UGX" },
];

export const kreatorsOptions = [
	{ value: "Kreator’s Dashboard", label: "Kreator’s Dashboard" },
	{ value: "Quick Stats", label: "Quick Stats" },
	{ value: "Add Product", label: "Add Product" },
	{ value: "Coupons", label: "Coupons" },
	{ value: "Affiliate Options", label: "Affiliate Options" },
	{ value: "Reports", label: "Reports" },
];

export const affiliateOptions = [
	// { value: "Affiliate Dashboard", label: "Affiliate Dashboard" },
	{ value: "/account/dashboard/affiliate", label: "Affiliate Dashboard" },
	{ value: "Quick Stats", label: "Quick Stats" },
	{ value: "Find Products", label: "Find Products" },
];
