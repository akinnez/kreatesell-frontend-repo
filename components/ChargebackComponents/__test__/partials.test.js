import {dayOptions, currencyOptions, ticketsOptions} from '../partials';

describe('DATA : ', () => {
	test('dayOptions data snapshots ', () => {
		expect(dayOptions).toMatchSnapshot();
	});

	test('currencyOptions data snapshots', () => {
		expect(currencyOptions).toMatchSnapshot();
	});
	test('ticketsOptions data snapshots', () => {
		expect(ticketsOptions).toMatchSnapshot();
	});
});
