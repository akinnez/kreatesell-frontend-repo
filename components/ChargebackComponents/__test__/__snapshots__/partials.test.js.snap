// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DATA :  currencyOptions data snapshots 1`] = `
Array [
  Object {
    "label": "NGN",
    "value": "Nigeria",
  },
  Object {
    "label": "USD",
    "value": "USA",
  },
  Object {
    "label": "GBP",
    "value": "Spain",
  },
  Object {
    "label": "KES",
    "value": "Spain",
  },
  Object {
    "label": "ZAR",
    "value": "Spain",
  },
  Object {
    "label": "TZX",
    "value": "Spain",
  },
  Object {
    "label": "UGX",
    "value": "Spain",
  },
]
`;

exports[`DATA :  dayOptions data snapshots  1`] = `
Array [
  Object {
    "label": "Today",
    "value": "Today",
  },
  Object {
    "label": "Yesterday",
    "unit": 1,
    "value": "Yesterday",
  },
  Object {
    "label": "Last 7 days",
    "unit": 7,
    "value": "Last 7 days",
  },
  Object {
    "label": "Last 30 days",
    "unit": 30,
    "value": "Last 30 days",
  },
  Object {
    "label": "This year",
    "unit": 365,
    "value": "This year",
  },
]
`;

exports[`DATA :  ticketsOptions data snapshots 1`] = `
Array [
  Object {
    "label": "All Tickets",
    "value": "All Tickets",
  },
  Object {
    "label": "Open Tickets",
    "value": "Opened Tickets",
  },
  Object {
    "label": "Closed Tickets",
    "value": "Closed Tickets",
  },
]
`;
